lib/egree/version.rb

```diff
@@ -1,3 +1,3 @@
 module Egree
-  VERSION = "0.0.6"
```

Comment:
I agree, lets just keep it in a separate branch (api/v2).
When it's ready, we can create a legacy branch called api/v1 from tag v0.0.6, 
merge api/v2 to master and bump the version.